composer.json

```diff
@@ -78,11 +78,11 @@
     },
     "require-dev": {
         "aws/aws-sdk-php": "^3.0",
-        "doctrine/dbal": "^2.6",
+        "doctrine/dbal": "^2.6|^3.0",
         "filp/whoops": "^2.4",
         "guzzlehttp/guzzle": "^6.3|^7.0",
         "league/flysystem-cached-adapter": "^1.0",
-        "mockery/mockery": "^1.3.1",
+        "mockery/mockery": "~1.3.2|^1.4.1",
```

Comment:
@GrahamCampbell why was the caret changed here?

Reply:
Because we want at least 1.3.2 or at least 1.4.1, but not 1.4.0.

Reply:
Gotcha 👍